secondary-server-2/src/main/java/com/replicated_log/secondary_server_2/service/SecondaryServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package com.replicated_log.secondary_server_2.service;
+
+import com.replicated_log.secondary_server_2.model.Item;
+import com.replicated_log.secondary_server_2.repository.ItemStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Set;
+
+@Service
+public class SecondaryServiceImpl implements SecondaryService {
+
+    @Autowired
+    private ItemStorage itemStorage;
```

Comment:
why we storage do not call as repository?

Reply:
classes ckrresponding to storages were renamed to repositories in the final iteration